visual_frontend/measurement_manager:
  plugins:
    - "visual_frontend::FeatureMotion"
    # - "visual_frontend::DifferenceImage"
    # - "visual_frontend::DarknetPlugin"
    # - "visual_frontend::ColorDetector"

visual_frontend/transform_manager:
  plugins:
    - "visual_frontend::SimpleHomography"

visual_frontend/feature_manager:
  plugins:
    - "visual_frontend::LKTTracker"
